src/core/utils/helpers/general.ts

```diff
@@ -415,7 +415,7 @@ export const patronAgeValid = (cpr: string, minAge: number) => {
 };
 
 export const constructModalId = (prefix: string, fragments: string[]) =>
-  `${prefix ? `${prefix}-` : ""}${fragments.join("-")}`;
+  `${prefix ? `${prefix}` : ""}${fragments.join("-")}`;
```

Comment:
I don't get this change. Why shouldn't the prefix be followed by a separator?
Actually I have come up with a more nice way to define this function:
```typescript
export const constructModalId = (prefix: string, fragments: string[]) =>
  `${(prefix ? [prefix, ...fragments] : fragments).join('-')}`;